### ISSUE TITLE
Train ensembles on top of models

### ISSUE THREAD

**Issue by MichalOleszak** [MichalOleszak/footballpredict#3]
- [x] build basic keras framework
- [x] enable predicting new data 
- [ ] tune model with dropout, regularization and layers
